setup.py

```diff
@@ -19,7 +19,7 @@
         "provider": [
             "scrapy-poet>=0.9.0",
             "web-poet>=0.13.0",
-            "zyte-common-items",
+            "zyte-common-items>=0.7.0",
```

Comment:
this is a bit tricky, because scrapy-zyte-api supports Python 3.7, but recent zyte-common-items don't. I think that's fine, because this dependency is optional.

Reply:
Should we just drop Python 3.7 support?

Reply:
Let's not be aggressive with deprecating old software here, to aid the migration from SPM to Zyte API.